--- terraform/services/.terraform/modules//aml.azureml/main.tf	2025-03-15 13:55:35.451091236 +0000
+++ patch/aml/main.tf	2025-03-15 14:18:23.258317820 +0000
@@ -37,7 +37,7 @@
     kind = var.kind
   }
   location  = var.location
-  name      = "aml-${var.name}"
+  name      = "${var.name}"
   parent_id = data.azurerm_resource_group.current.id
   replace_triggers_external_values = [
     var.resource_group_name # since this is the value that determines if parent_id changes, require create/destroy if it changes
